config-secret/api-server/src/main/resources/application.properties

```diff
@@ -6,3 +6,4 @@ quarkus.s2i.base-jvm-image=registry.access.redhat.com/openjdk/openjdk-11-rhel7
 
 %test.hello.message=Hello, %s!
 %test.quarkus.kubernetes-config.enabled=false
+%test.quarkus.kubernetes-config.secrets.enabled=false
```

Comment:
Add TODO to remove this once https://github.com/quarkusio/quarkus/issues/11968 gets fixed

Reply:
Done.